packages/EasyErrorHandler/src/Reporters/AbstractErrorReporter.php

```diff
@@ -6,23 +6,17 @@
 
 use EonX\EasyErrorHandler\Interfaces\ErrorLogLevelResolverInterface;
 use EonX\EasyErrorHandler\Interfaces\ErrorReporterInterface;
+use EonX\EasyUtils\Traits\HasPriorityTrait;
 
 abstract class AbstractErrorReporter implements ErrorReporterInterface
 {
-    /**
-     * @var \EonX\EasyErrorHandler\Interfaces\ErrorLogLevelResolverInterface
-     */
-    protected $errorLogLevelResolver;
+    use HasPriorityTrait;
 
-    /**
-     * @var int
-     */
-    private $priority;
-
-    public function __construct(ErrorLogLevelResolverInterface $errorLogLevelResolver, ?int $priority = null)
-    {
-        $this->errorLogLevelResolver = $errorLogLevelResolver;
-        $this->priority = $priority ?? 0;
+    public function __construct(
+        protected readonly ErrorLogLevelResolverInterface $errorLogLevelResolver,
+        ?int $priority = null
+    ) {
+        $this->doSetPriority($priority);
     }
 
     public function getPriority(): int
```

Comment:
Can we remove this method? It's already defined in `HasPriorityTrait`.
The same for `AbstractErrorReporter`.

Reply:
Thanks. done.